platform/stm32/templates/f1/stm32f103_bluepill/build.conf

```diff
@@ -0,0 +1,14 @@
+TARGET = embox
+
+PLATFORM = stm32f103_bluepill
+
+ARCH = arm
+
+CROSS_COMPILE = arm-none-eabi-
+
+CFLAGS += -O0 -g
+CFLAGS += -mthumb -mlittle-endian
+CFLAGS += -march=armv7-m -mtune=cortex-m3
+CFLAGS += -ffreestanding
+
+CFLAGS += -DSTM32F100xB
```

Comment:
It's better to use `@BuildArtefactPath` for `platform.stm32.f1.stm32f103_bluepill.stm32f103_bluepill_conf` module for this purpose